Department-for-Transport-Disruptions/create-disruptions-data
# feat: Now button on create disruption page

To test:

Go to the create disruption page and then scroll to publication date and time
As per wireframes should be a 'Now' button click on it to see the current date and time
